app/src/main/java/com/example/kursakademiaandroida/core/api/model/ResponseInfo.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.kursakademiaandroida.core.api.model
+
+import com.google.gson.annotations.SerializedName
+
+
+data class ResponseInfo(
+    @SerializedName("count") val count: Int,
+    @SerializedName("pages") val pages: Int,
+    @SerializedName("next") val next: String?,
+    @SerializedName("prev") val prev: String
```

Comment:
Jeśli pobieramy pierwszy "page" to wtedy to pole będzie nullowe. Warto oznaczyć je jako opcjonalne.